README.md

```diff
@@ -82,34 +53,68 @@ In order to push to DockerHub, you must set the following ```secrets``` in your
 # make sure you are logged into Azure
 az account show
 
-# if not log in
+# if not, log in
 az login
 
 ```
 
+### Verify Key Vault Access
+
+```bash
+
+# verify you have access to Key Vault
+az keyvault secret show --name CosmosDatabase --vault-name $He_Name
+
+```
+
 ### Run the app
 
+> If you are using CodeSpaces, you just need to set the --keyvault-name parameter and press F5
+
```

Comment:
Maybe call out where to set this in the .vscode file

Reply:
Good catch - I updated here and also added the parameters to the /docs/vscode-template/launch.json file